docs/cugraph/source/api_docs/index.rst

```diff
@@ -1,16 +1,39 @@
-Python API reference
-====================
+API Reference
+=============
 
 This page provides a list of all publicly accessible Python modules with in the Graph collection
 
+Core Graph API Documentation
+----------------------------
+
 .. toctree::
-    :maxdepth: 2
-    :caption: Python API Documentation
+    :maxdepth: 3
+    :caption: Core Graph API Documentation
 
     cugraph/index.rst
     plc/pylibcugraph.rst
+    cugraph_c/index.rst
+    cugraph_cpp/index.rst
+
+Graph Nerual Networks API Documentation
+---------------------------------------
+
+.. toctree::
+    :maxdepth: 3
+    :caption: Graph Nerual Networks API Documentation
+
     cugraph-dgl/cugraph_dgl.rst
     cugraph-pyg/cugraph_pyg.rst
-    service/index.rst
     cugraph-ops/index.rst
+    wholegraph/index.rst
+
+Graph Extensions API Documentation
```

Comment:
I wonder if there's a better name for this category.  No action for this PR, but I didn't expect to see cugraph-service under Graph Extensions.